packages/salesforcedx-apex-replay-debugger/test/integration/adapter.test.ts

```diff
@@ -49,7 +49,7 @@ describe('Replay debugger adapter - integration', () => {
       'apex-replay'
     );
     await dc.start();
-    dc.defaultTimeout = 10000;
+    dc.defaultTimeout = 15000;
```

Comment:
Why does the defaultTimeout need to be increased in this unigration test?

Reply:
I was just trying some things and didn't realized this got pushed. I reverted this change